Libplanet.Net/Consensus/ConsensusReactor.cs

```diff
@@ -24,22 +24,30 @@ public class ConsensusReactor<T> : IReactor
 
         private readonly Codec _codec = new Codec();
 
-        private RoutingTable _routingTable;
-        private ITransport _transport;
+        private RoutingTable _blockRoutingTable;
```

Comment:
This is not a just `Block` so IMHO this is ok to left `routingTable` and `transport`